src/stories/HeatmapVis.stories.tsx

```diff
@@ -108,12 +108,25 @@ export const CustomCoordinates = Template.bind({});
 CustomCoordinates.args = {
   dataArray,
   domain,
-  abscissas: Array(dataArray.shape[1] + 1)
-    .fill(0)
-    .map((x, i) => 100 + 10 * i),
-  ordinates: Array(dataArray.shape[0] + 1)
-    .fill(0)
-    .map((x, i) => -5 + 0.5 * i),
+  abscissaParams: {
+    values: Array(dataArray.shape[1])
```

Comment:
I now pass datasets of size `n` (and not `n+1`) to test the extension of custom coordinates.

Reply:
Should we test both?

Reply:
Yes, I think so !